Gemfile

```diff
@@ -173,7 +173,7 @@ end
 group :production do
   # sidekiq enterprise requires a license key to download but is only required in production.
   # for local dev environments, regular sidekiq works fine
-  unless ENV['EXCLUDE_SIDEKIQ_ENTERPRISE'] == 'true'
+  unless Bundler::Settings.new['enterprise.contribsys.com']&.empty? && ENV['BUNDLE_ENTERPRISE__CONTRIBSYS__COM']&.empty?
```

Comment:
Rails isn't loaded at this point, so I can't use `present?` or `blank?`.

Reply:
for the ENV, i'd recommend `ENV.fetch('BUNDLE_ENTERPRISE__CONTRIBSYS__COM', false)` -- that way you're ensuring a default

Reply:
If someone is running this natively and `ENV['BUNDLE_ENTERPRISE__CONTRIBSYS__COM']` is not set, it will be `nil`. 
However, if using Docker and it's not set, Docker defaults the value to an empty string `''`.
I do not know of a way to have Docker default a variable value to "not set".  Do you?

So, I could do `ENV.fetch('BUNDLE_ENTERPRISE__CONTRIBSYS__COM', '').empty?`